torchao/quantization/quant_api.py

```diff
@@ -939,6 +940,9 @@ def float8_dynamic_activation_float8_weight(
         mm_config (Float8MMConfig): Configuration for the matrix multiplication. Default uses fast accumulation.
 
     """
+    assert (
+        is_cuda_8_9
+    ), "Float8 dynamic activation quantization is only supported on CUDA 8.9 and above"
```

Comment:
This should also be supported on AMD. We should probably update this check.

cc @jeffdaily